tests/foreman/cli/test_classparameters.py

```diff
@@ -16,107 +16,253 @@
+        })
+        self.assertEqual(sc_param['override'], True)
+        Host.update({
+            u'name': self.host_name,
+            u'puppet-classes': 'ntp'
+        })
+        host_sc_params_list = Host.sc_params({u'host': self.host_name})
+        self.assertGreater(len(host_sc_params_list), 0)
 
     @run_only_on('sat')
-    @stubbed()
-    @tier3
+    @tier2
     def test_positive_list_parameters_by_host_id(self):
         """List all the parameters included in specific Host by its id.
 
         @id: 79050de6-b894-4a88-b155-32bf488b692c
 
         @assert: Parameters listed for specific Host.
 
-        @caseautomation: notautomated
-
-        @CaseLevel: System
+        @CaseLevel: Integration
         """
+        sc_param_id = self.sc_params_ids_list.pop()
+        SmartClassParameter.update({
+            'id': sc_param_id,
+            'override': 1,
+        })
+        sc_param = SmartClassParameter.info({
+            'puppet-class': 'ntp',
+            'id': sc_param_id,
+        })
+        self.assertEqual(sc_param['override'], True)
+        Host.update({
+            u'name': self.host_name,
+            u'puppet-classes': 'ntp'
+        })
+        host_id = Host.info({u'name': self.host_name})['id']
+        host_sc_params_list = Host.sc_params({u'host-id': host_id})
+        self.assertGreater(len(host_sc_params_list), 0)
 
     @run_only_on('sat')
-    @stubbed()
-    @tier3
+    @tier2
     def test_positive_list_parameters_by_hostgroup_name(self):
         """List all the parameters included in specific HostGroup by its name.
 
         @id: a2a01ca7-4dd2-4db6-a654-a632864998d9
 
         @assert: Parameters listed for specific HostGroup.
 
-        @caseautomation: notautomated
-
-        @CaseLevel: System
+        @CaseLevel: Integration
         """
+        sc_param_id = self.sc_params_ids_list.pop()
+        SmartClassParameter.update({
+            'id': sc_param_id,
+            'override': 1,
+        })
+        sc_param = SmartClassParameter.info({
+            'puppet-class': 'ntp',
+            'id': sc_param_id,
+        })
+        self.assertEqual(sc_param['override'], True)
+        hostgroup = make_hostgroup({
+            'environment-id': self.env['id'],
+            'puppet-class-ids': self.puppet['id']
+        })
+        hostgroup_sc_params = HostGroup.sc_params({
+            u'hostgroup': hostgroup['name']})
+        self.assertGreater(len(hostgroup_sc_params), 0)
 
     @run_only_on('sat')
-    @stubbed()
-    @tier3
+    @tier2
     def test_positive_list_parameters_by_hostgroup_id(self):
         """List all the parameters included in specific HostGroup by id.
 
         @id: 80c1058d-b87d-4c09-957f-7d3daacdedf4
 
         @assert: Parameters listed for specific HostGroup.
 
-        @caseautomation: notautomated
-
-        @CaseLevel: System
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier3
-    def test_positive_list_parameters_by_puppetclass_name(self):
```

Comment:
Because, they were not actual, because it was not possible to use puppet class for list command, but I can, so will return these two test cases in a minute.
Of course, they are covered explicitly in setup class, but let's add them anyway